ext/v8/object-template.h

```diff
@@ -43,6 +55,88 @@ namespace rr {
 
       return Qnil;
     }
+
+    static VALUE SetAccessor(int argc, VALUE argv[], VALUE self) {
```

Comment:
I'm a bit confused about which method we're going to call here and why. The way it is written it looks like it will invoke the C++ method that takes `Local<Name>` instead of `Local<String>` This seems fair since string is always a name, but will it make a difference at runtime? I.e. is there a different codepath that V8 is executing?